rclpy/src/rclpy/_rclpy.c

```diff
@@ -666,6 +673,7 @@ rclpy_create_node(PyObject * Py_UNUSED(self), PyObject * args)
   rcl_node_options_t options = rcl_node_get_default_options();
   options.use_global_arguments = use_global_arguments;
   options.arguments = arguments;
+  options.enable_rosout = enable_rosout;
```

Comment:
@suab321321 what is the function of this change?

Reply:
@allenh1 I dont know sir what is the use of this change at first I committed the mistake of pushing to this PR for custom exception then afterwards I made the _rclpy.c same as that was before(by copying the _rclpy.c from ros2/rclpy master branch) so that these changes wont be reflected here.

Reply:
> by copying the _rclpy.c from ros2/rclpy master branch) so that these changes wont be reflected here...

@suab321321 that explains it... Try this to reset the file for your current branch.

```
git checkout origin/master -- rclpy/src/rclpy/_rclpy.c
git commit -s -m "Reset _rclpy.c to remove unrelated changes"
```

Then let's go ahead and rebase your changes on the `ros2` master.

```
git remote add ros2 https://github.com/ros2/rclpy
git fetch ros2
git rebase ros2/master
git push origin master --force-with-lease
```

Reply:
@allenh1 sir after pushing it is showing that everything is up-to-date

Reply:
@allenh1 sir local repo are showing old codes now.